app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ActiveRecord::Base
+  validates :lat, presence: true
```

Comment:
@edwardloveall do we have a preference in the style guide for

`validates_presence_of` vs `validates ... presence: true`

I tend to use `validates_presence_of` as it confirm closer to the shoulda_matchers.

Thoughts?